### DISCUSSION TITLE
Clear cache CLI command

### DISCUSSION THREAD

**Discussion by Dominic-Marcelino** [directus/directus#18815] [Feature Requests]
### Summary

Before [v9.25.2](https://github.com/directus/directus/releases/tag/v9.25.2) directus used to clear the cache on startup. This has changed for better support in horizontal scaling https://github.com/directus/directus/pull/18238.

As in some Environments it still makes sense to clear the automatically on startup or after deployment I'd suggest to add a CLI command that can be run in CI.

_I know that there's a dedicated endpoint but a command would be easier to add it to a CI/CD pipeline._

### Motivation

Clear the cache after each deployment.

### Detailed Design

Add a new command to clear the cache:

`'npx directus cache clear`

### Must Have:
- CLI command

### Drawbacks

n.A

### Alternatives

n.A

### Adoption Strategy

n.A

**Comment by rijkvanzanten** [comment]
Heya! Thanks for opening this feature request!

This feature request has received over **15 votes** from the community. This means we'll move this feature request to the Under Review state! 

The Core team will schedule a meeting to review this request as soon as possible. The discussion will then be approved or denied. You may or may not be invited to join this meeting with the core team.

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
